records/generator.go

```diff
@@ -554,6 +561,10 @@ func (rg *RecordGenerator) taskContextRecord(ctx context, task state.Task, f sta
 	rg.insertTaskRR(arec+tail, ctx.taskIP, A, enumTask)
 	rg.insertTaskRR(canonical+tail, ctx.taskIP, A, enumTask)
 
+	// insert PTR records, these are only served when ReverseDnsOn is true,
+	// which sets up a handler for the reveverse DNS zones
+	rg.insertTaskRR(arec+tail, ctx.taskIP, PTR, enumTask)
```

Comment:
@christian-marie Perhaps we should only insert this type when configured to do so? Wouldn't want unusable data taking up space.

Reply:
Don't really care about performance or this patch anymore.